### PR TITLE
Add talk to an event

### PR DESCRIPTION
**Pull request by akrabat**
Add the ability to add a talk to an event. 

This PR also implements the yellow admin bar for the event section as we now have three items for it and will have more, so this seemed a good time to to do it.

The ability to add speakers when adding the talk is also implemented. This works the same way as web1 and depends on [API PR #320](https://github.com/joindin/joindin-api/pull/320).

In order for the talk to be added to the selected track, [API PR #328](https://github.com/joindin/joindin-api/pull/328) is required.

### CODE REVIEW

**Comment by akrabat** [comment]
hmm... I've just realised that there's no way to add a talk to a track when creating it. 

@lornajane: How do you want to attach a talk to a track?

**Comment by lornajane** [comment]
Since we can't create tracks, this is probably out of scope this time
around?

**Comment by lornajane** [comment]
This PR is now showing conflicts, can you have a quick look?

**Comment by akrabat** [comment]
Rebased now that #331 has been merged.

File: app/src/Event/EventController.php [hunk 1]
```diff
@@ -606,15 +612,15 @@ public function redirectFromId($eventId, $extra = false)
      */
     private function addEventUsingForm(Form $form)
     {
-        $eventApi = $this->getEventApi();
-        $values = $form->getData();
+        $eventapi = $this->geteventapi();
```
Comment (Jean85) [inline app/src/Event/EventController.php]: What are these changes? Unwanted "Toggle caps"?

**Comment by Jean85** [comment]
LGTM.

I've tested this locally and it seems to work. 
The only quirk that I found was the timepicker helper in the form: if I click on the field, the picker comes up with a prefilled value (e.g. 9:00AM) but the field itself stays empty until I change it; so, if the prefilled value was ok for me, I have to move the time up\down and then back to the prefilled value to make it work.

File: app/src/Event/EventController.php (hunk 1 above)
Comment (akrabat) [inline app/src/Event/EventController.php]: Good catch. Fixed.

**Comment by liam-wiltshire** [comment]
I'm officially on holiday now (whooo!), so I'll try to check and merge some of these larger PRs this week.

**Comment by akrabat** [comment]
Rebased and ran `composer dumpautoload` in order to pick up and update `autoload_static.php`.

**Pull request merged**